physlr/physlr.py

```diff
@@ -1500,9 +1506,6 @@ def detect_communities_cosine_of_squared(g, node_set, squaring=True, threshold=0
         Square the adjacency matrix and then use cosine similarity to detect communities.
         Return communities.
         """
-        import scipy as sp
```

Comment:
I guess the good thing about having the import statements like this was that the user wouldn't require those packages unless they were using this specific molecule separation mode?

What was the pylint error here? What do we think about following the error vs. editing the config file to ignore such issues?

Reply:
```
physlr/physlr.py:1492:8: C0415: Import outside toplevel (community) (import-outside-toplevel)
physlr/physlr.py:1503:8: C0415: Import outside toplevel (scipy) (import-outside-toplevel)
physlr/physlr.py:1504:8: C0415: Import outside toplevel (numpy) (import-outside-toplevel)
physlr/physlr.py:1811:8: C0415: Import outside toplevel (physlr.mkt) (import-outside-toplevel)
physlr/physlr.py:1812:8: C0415: Import outside toplevel (numpy) (import-outside-toplevel)
```

Reply:
might able to disable that warning specifically. I dont know

Reply:
If we want to, you can add an exception in the `.pylintrc`

Reply:
Done